zerver/models.py

```diff
@@ -456,6 +456,14 @@ def get_signup_notifications_stream(self) -> Optional['Stream']:
             return self.signup_notifications_stream
         return None
 
+    def has_only_one_admin(self) -> bool:
+        if len(UserProfile.objects.filter(realm=self, is_bot=False,
```

Comment:
ok sure

Reply:
sorry I couldn't get you

Reply:
@punchagan

Reply:
Instead of writing this, 
```python
if condition:
    return False
else:
    return True
```
you could, say

```python
return not condition
# or return inverted_condition
```

Reply:
ok sure